examples/normalizer/Normalize.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
```

Comment:
Shebangs are always tricky.

Some people may not have python in `/usr/bin` or may be using a virtualenv, also this isn't py3 compatible and `env python` often defaults to py3.

Would recommend:

```bash
#!/usr/bin/env python2
```